server/generic_server.cc

```diff
@@ -6,11 +6,11 @@
 namespace ctk = ChimeraTK;
 
 struct GenericApp : public ctk::Application {
-  GenericApp() : Application("generic_server") {
+  GenericApp() : Application("generic_device_server") {
     ChimeraTK::setDMapFilePath("devices.dmap");
   }
   ~GenericApp() { shutdown(); }
-  ctk::ConfigReader config{this, "Config", getName() + "_configuration.xml", {}};
```

Comment:
This produces generic_server_configuration.xml. I am wondering how it worked with doocs-generic-chimeratk_server.xml in the first place. Please check with @nshehz to make this consistent.